charts/pega/values.yaml

```diff
@@ -50,17 +50,17 @@ global:
     customerDataSchema: ""
 
   customArtifactory:
-    # If using a custom artifactory to download JDBC uri, supply the authentication details here.
-    auth:
+    # If using a custom artifactory secured with Basic or APIKey Authentication to download JDBC driver, supply the authentication details here.
```

Comment:
If you use a secured custom artifactory to manager your JDBC driver, provide the authentication details below by filling in the appropriate authentication section, either basic or apiKey.